plenum/test/view_change_with_delays/test_view_change_with_delayed_commits_on_multiple_nodes_and_restarts_of_those_nodes.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from plenum.test.helper import perf_monitor_disabled
+from plenum.test.view_change_with_delays.helper import do_view_change_with_delayed_commits_and_node_restarts
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    """
+    Patch config so that monitor won't start view change unexpectedly
+    """
+    with perf_monitor_disabled(tconf):
+        yield tconf
+
+
+"""
+Not sure what is happening with this test. Locally it is almost always green (saw it fail just a couple of times).
+On Jenkins it fails regularly with the following error:
+def checkViewNoForNodes(nodes: Iterable[TestNode], expectedViewNo: int = None):
+        viewNos = set()
+        for node in nodes:
+            logger.debug("{}'s view no is {}".format(node, node.master_replica.viewNo))
+            viewNos.add(node.master_replica.viewNo)
+        assert len(viewNos) == 1, 'Expected 1, but got {}. ' \
+>                                 'ViewNos: {}'.format(len(viewNos), [(n.name, n.master_replica.viewNo) for n in nodes])
+E       AssertionError: Expected 1, but got 2. ViewNos: [('Alpha', 1), ('Beta', 1), ('Gamma', 0), ('Delta', 0)]
+
+expectedViewNo = 1
+node       = Delta
+nodes      = [Alpha, Beta, Gamma, Delta]
+viewNos    = {0, 1}
+"""
+@pytest.mark.skip(reason="Maybe will be fixed by: INDY-2238 (Persist 3PC messages during Ordering)")
```

Comment:
I don't see reasons why it should not pass. Slow nodes need to catchup txns from the fast ones (there are F+1 fast nodes which should be enough). And they all should be able to finish view change.
Maybe the reason is that nodes try to start view change before they finish catch up. You can try to add `ensure_all_nodes_have_same_data` in `do_view_change_with_delayed_commits_and_node_restarts` before doing view change.